extensions/resource-deployment/src/ui/resourceTypeWizard.ts

```diff
@@ -59,12 +62,26 @@ export class ResourceTypeWizard {
 		} else if (instanceOfAzureSQLDBDeploymentProvider(this.provider)) {
 			return new DeployAzureSQLDBWizardModel(this.provider, this);
 		}
-		// other types are undefined for now.
-		return undefined;
+		return new PageLessDeploymentModel(this.provider, this);
 	}
 
 	public async open(): Promise<void> {
-		this.model.initialize();
+		this.wizardObject = azdata.window.createWizard(this.resourceType.displayName, this.resourceType.name, 'wide');
+		this.model = this.getResourceProviderModel();
+		await this.wizardObject.open();
+	}
+
+	public async refresh(): Promise<void> {
```

Comment:
Having something named "refresh" closing and re-opening a dialog isn't what I would expect. What's the use case for this anyways?

Reply:
I'm guessing this is related to the PR description about the titles and listeners. Please add those details here and make sure to file an issue to get that cleaned up - this is very much not ideal.

(I would also suggest that you instead call close and open in turn instead of making this function - that would at least clearly indicate what's happening to the caller)